src/js/vertical-slide.js

```diff
@@ -6,15 +6,28 @@ const shortCodeSlug = sliderContext.getAttribute('data-shortcode-slug');
 
 // Fetching the data with id
 const fetchSliderData = async (id, slug) => {
-  const response = await fetch(`http://localhost/wp-local/wp-json/wp/v2/vertical_slider/${id}?_fields=id,acf`);
-  const data = await response.json();
-  return data;
+  if (id) {
+    const response = await fetch(`http://localhost/wp-local/wp-json/wp/v2/vertical_slider/${id}?_fields=custom_rest_api`);
+    const data = await response.json();
+    return data;
```

Comment:
What if this returns a 404? Is there any error handling we need to add here? Will this just return null and the error be handled elsewhere?